### DISCUSSION TITLE
I have started docker, but there is no username and password

### DISCUSSION THREAD

**Discussion by dahualian** [OpenAS2/OpenAs2App#355] [Q&A]
I have started docker, but there is no username and password

**Comment by igwtech** [comment]
Default username and password are defined in the config.xml file. Current
default values are:
username: userID
password: pWd

Note; mind upper/lower casing.

Regards, J

On Tue, Feb 6, 2024 at 6:30 AM dahualian ***@***.***> wrote:

> I have started docker, but there is no username and password
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenAS2/OpenAs2App/discussions/355>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA2QND2IA4ICF4OIQGFXA7LYSIA3ZAVCNFSM6AAAAABC3W6UEKVHI2DSMVQWIX3LMV43ERDJONRXK43TNFXW4OZWGE4DKOBZGI>
> .
> You are receiving this because you are subscribed to this thread.Message
> ID: ***@***.***>
>


-- 
Javier Munoz

CEO

Greicodex Software



+58 2127629120 <+58+2127629120>
***@***.***
www.greicodex.com
Av Francisco Solano, Centro Solano Plaza, Oficina PHA, Caracas, Venezuela

**Reply by dahualian** [reply]
Thank you, But Still can't log in


![20240207092214](https://github.com/OpenAS2/OpenAs2App/assets/14847891/28910eae-f7ac-4536-bba5-b0e31105c706)

**Comment by igwtech** [comment]
Hi, that network error means your API URL is not correct or the container
doesn't have the port properly exported


On Tue, Feb 6, 2024 at 9:23 PM dahualian ***@***.***> wrote:

> Thank you, But Still can't log in
>
> 20240207092214.jpg (view on web)
> <https://github.com/OpenAS2/OpenAs2App/assets/14847891/28910eae-f7ac-4536-bba5-b0e31105c706>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenAS2/OpenAs2App/discussions/355#discussioncomment-8389844>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA2QNDY4NEBISJ7XWRLZGX3YSLJQZAVCNFSM6AAAAABC3W6UEKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGOBZHA2DI>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Javier Munoz

CEO

Greicodex Software



+58 2127629120 <+58+2127629120>
***@***.***
www.greicodex.com
Av Francisco Solano, Centro Solano Plaza, Oficina PHA, Caracas, Venezuela

**Comment by mohsin-ansari** [comment]
Facing the same network error & ports are properly configured

![image](https://github.com/OpenAS2/OpenAs2App/assets/46895944/1d0e4dd7-b52d-4749-a3e4-12f4040db04d)

**Reply by uhurusurfa** [reply]
Trying acccessing the API directly from the command line on the server using curl

**Reply by mohsin-ansari** [reply]
Tried curl on command like but getting error "empty response from server" and "socket hung up" on postman.

When I am running webUI and openas2 server separately without docker or docker compose both are running fine and able to login. Do docker compose file need some modifications ?

**Reply by mohsin-ansari** [reply]
Done everything but still no success with docker, sometimes it works and sometime not, had to run without docker.
Now each time getting different error when trying to do directory pooling. Files are getting polled but getting some issues with 'msg ' variable which is also not clear in the how to document and where to set these ?

![image](https://github.com/user-attachments/assets/00e08aa5-1d4f-4671-8b7b-f30f8f69ad5e)

**Comment by igwtech** [comment]
If the connection is closing without response then your docker exported
port is not working properly, meaning you exported a port on the Docker
Framework on the HOST that doesn't have a corresponding listening daemon on
the Container. Please verify the documentation on the Readme of the
container on how to use the docker command line parameters to run the
container and override the XML configuration parameters including, port,
username and password with environment variables.

On Tue, Jul 9, 2024 at 9:33 AM mohsin-ansari ***@***.***>
wrote:

> Tried curl on command like but getting error "empty response from server"
> and "socket hung up" on postman.
>
> When I am running webUI and openas2 server separately without docker or
> docker compose both are running fine and able to login. Do docker compose
> file need some modifications ?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/OpenAS2/OpenAs2App/discussions/355#discussioncomment-9998700>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA2QND5NDG3AJVVHCZJ26EDZLPRC5AVCNFSM6AAAAABC3W6UEKVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4TSOJYG4YDA>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>


-- 
Javier Munoz

CEO

Greicodex Software



+58 2127629120 <+58+2127629120>
***@***.***
www.greicodex.com
Av Francisco Solano, Centro Solano Plaza, Oficina PHA, Caracas, Venezuela

**Reply by mohsin-ansari** [reply]
Done everything but still no success with docker, sometimes it works and sometime not, had to run without docker.
Now each time getting different error when trying to do directory pooling. Files are getting polled but getting some issues with 'msg ' variable which is also not clear in the how to document and where to set these ?


![image](https://github.com/user-attachments/assets/664eff7a-6dcd-4b1f-a429-78a615563e10)

**Reply by uhurusurfa** [reply]
To fix this werror you must make sure that all the parameter values in the "pollerConfigBase" element in your config.xml or the "pollerConfig" elements in your partnerships.xml do NOT use the the "msg dynamic variable structure (`$msg.xxx.yyy$ `). Instead of `$msg.sender.as2_id$` use `$partnership.sender.as2_id$ `
